app/controllers/alchemy/api/pages_controller.rb

```diff
@@ -5,11 +5,14 @@ class Api::PagesController < Api::BaseController
     # Returns all pages as json object
     #
     def index
-      @pages = Page.accessible_by(current_ability, :index)
```

Comment:
`Page.accessible_by` always raises an error due to the way the `Permissions` are written. Not sure if this method was used by anybody